index.js

```diff
@@ -256,7 +266,8 @@ async function installMicromamba (inputs, extraChannels) {
 
 function isSelected (item) {
   if (/sel\(.*\):.*/gi.test(item)) {
-    return new RegExp('sel\\(' + MAMBA_PLATFORM + '\\):.*', 'gi').test(item)
+    const condaPlatform = getCondaArch().split('-')[0]
+    return new RegExp(`sel\\(${condaPlatform}\\):.*`, 'gi').test(item)
```

Comment:
@wolfv does Mamba support CPU arch specific selectors like `sel(linux-aarch64)` or only `sel(linux)`?

Reply:
only `sel(linux)` at the moment, sadly.